beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/path/aws/S3BytesDeletedReporter.java

```diff
@@ -37,7 +37,7 @@
   private final Map<String, Long> keyToSize = new HashMap<>();
   private final Counter counter;
 
-  public S3BytesDeletedReporter(S3Client s3Client, MeterRegistry meterRegistry, boolean dryRunEnabled) {
+  public S3BytesDeletedReporter(S3Client s3Client, GraphiteMeterRegistry meterRegistry, boolean dryRunEnabled) {
```

Comment:
Just checking, these don't work with the more generic type? i.e. if you make the change in the beans to be `GraphiteMeterRegistry` can't these stay as just `MeterRegistry` in the classes which use them? I can see how in the config you want to go for a specific type but if you make this change here then these classes can't be used with any other type of `MeterRegistry` which feels like a shame to introduce such coupling.

Reply:
Yep good point, I don't think these necessarily have to change. Will revert them back. Done.